src/site/content/en/blog/betty-crocker/index.md

```diff
@@ -0,0 +1,341 @@
+---
+title: |
+  Wake Lock API case study: 300% increase in purchase intent indicators on BettyCrocker.com
+subhead: |
+  After they have implemented the Wake Lock API, cooking site BettyCrocker.com's
+  indicators of purchase intent were about 300% higher for wake lock users compared to all users.
```

Comment:
```suggestion
  Nothing is worse when cooking with a mobile device than its screen turning off in the middle of a recipe step.
  Learn how cooking site BettyCrocker.com used the Wake Lock API for profit to prevent this from happening.
```